qiskit/circuit/__init__.py

```diff
@@ -780,6 +780,28 @@
 
 .. _circuit-custom-gates:
 
+
+Investigating commutation relations
+-----------------------------------
```

Comment:
This is moving the hyperlink target `circuit-custom-gates` to point at this new header, rather than the "Created custom instructions" header below.

Reply:
whoops -- fixed in [d01e891](https://github.com/Qiskit/qiskit/pull/13303/commits/d01e891f3ba5638849e80b93cbfcd50995dddd3b), thanks!